.asf.yaml

```diff
@@ -0,0 +1,17 @@
+github:
```

Comment:
If the metadata for the website, description, etc. follows the repo, we should be fine, but if not, we should add that too.

Reply:
Same as previous comment